src/main/java/com/carrotcreative/cream/strategies/generic/CacheStrategy.java

```diff
@@ -6,14 +6,14 @@
 
 import java.io.Serializable;
 
-public interface CacheStrategy<T extends LoaderParams> {
+public interface CacheStrategy<T extends LoaderParams, C extends Serializable> {
```

Comment:
I see that I used T here before.  You know I prefer descriptive names for generics, so maybe we could change these both to `Identifier` and `Content `

@roideuniverse what's your opinion about using descriptive names for generics?  I somehow remember you preferred letters, as it made it obvious it was a generic.  Curious of your thoughts now that you've dealt with some code with descriptive generic names!